### PR TITLE
[easy] stylistic cleanup of physical_coordinate_calculator.py

### PR DESCRIPTION
**Pull request by ttung**
1. `physcial` -> `physical`.
2. Reorder min before max for `_calculate_physical_pixel_size`.
3. Small line wrapping tweaks.
4. Fix annotations and docblock for `_recalculate_physical_coordinate_ranges`.

### CODE REVIEW

File: starfish/imagestack/physical_coordinate_calculator.py [hunk 1]
```diff
@@ -121,11 +123,12 @@ def _pixel_offset_to_physical_coordinate(physical_pixel_size: Number,
...
-                                           coord_max: float,
-                                           dimension_size: int,
-                                           indexer
-                                           ) -> Tuple[Number, Number]:
+def _recalculate_physical_coordinate_range(
+        coord_min: float,
+        coord_max: float,
+        dimension_size: int,
+        indexer
+) -> Tuple[Number, Number]:
```
Comment (shanaxel42) [inline starfish/imagestack/physical_coordinate_calculator.py]: So if there's listed params like this I should always start them on the next line?

File: starfish/imagestack/physical_coordinate_calculator.py [hunk 2]
```diff
@@ -42,8 +44,8 @@ def calc_new_physical_coords_array(physical_coordinates: xr.DataArray,
     return new_coords
 
 
-def _needs_coords_recalculating(x_indexers: Union[int, slice], y_indexers: Union[int, slice]
-                                ) -> bool:
+def _needs_coords_recalculating(
+        x_indexers: Union[int, slice], y_indexers: Union[int, slice]) -> bool:
     if isinstance(x_indexers, int) or isinstance(y_indexers, int):
```
Comment (shanaxel42) [inline starfish/imagestack/physical_coordinate_calculator.py]: If the return definition goes over the line limit you'd prefer to just have everything on the next line?

Comment (ttung) [reply starfish/imagestack/physical_coordinate_calculator.py]: Yup.  The rationale is the same: it minimizes LOC churn if we rename methods.

File: starfish/imagestack/physical_coordinate_calculator.py (hunk 1 above)
Comment (ttung) [reply starfish/imagestack/physical_coordinate_calculator.py]: So my preference is that for multiline params (calls and method declarations), that we start on the next line (basically the second option of https://www.python.org/dev/peps/pep-0008/#indentation).

Rationale: If the method name changes, we do not need to change the indentation for every line.

**Pull request merged**